VCFv4.4.tex

```diff
@@ -625,6 +625,20 @@ \section{INFO keys used for structural variants}
 \end{verbatim}
 \normalsize
 
+\footnotesize
+\begin{verbatim}
+##INFO=<ID=SVCLAIM,Number=1,Type=String,Description="Claim made by the structural variant call. Valid values are CN, BP, CNBP for copy number, breakpoint and both respectively.">
+\end{verbatim}
+\normalsize
+
+This field disambuates the claim being made by variant caller for DEL, INS, DUP structural variants.
+CN indicates the claim is a copy number claim and the call has been made based on a measure of DNA abundance of the called region.
+BP indicates the claim is a breakpoint claim and the call has been made based on the detection of a non-reference DNA adjacency.
+CNBP indicates the claim is both a copy number and breakpoint claim.
+Records with SVTYPE of CNV are implicitly copy number claims. SVCLAIM is optional for these records but, if specified, cannot be BP.
+Records with SVTYPE of BND or INV are implicitly breakpoint claims. SVCLAIM is optional for these records but, if specified, cannot be CN.
```

Comment:
> if specified, cannot be CN

does this prevent to state a CNBP claim with breakend notation? how is it expected to state for instance that a translocation has CN=1? (in the sense that it's not a deletion nor a duplication, but just a sequence that moved to another place)

Reply:
It's not intended to. I'll reword it to make this explicit.

Reply:
I think this CN (copy number claim) is meant to be different from the other CN (number of copies) which is the INFO/FORMAT tag. Actually, maybe we want to change _this_ CN to something else to avoid the confusion?

Reply:
That's a good point.  Would changing the nomenclature from copy number -> abundance and breakpoint -> novel adjacency distinguish it better?

Reply:
“Abundance" + “adjacency” sounds nice, although I'm not sure how to best abbreviate those. Presumably we need two/three letter acronyms, plus something to refer to the "both claims at once" case

Reply:
@d-cameron @jmmut @lbergelson 

> Abundance and adjacency are nice as they're clear and not overloaded. ADJ is obvious. Abundance not so much. AB, ABD, ABU? Would quantity/QTY work or is that going too far towards a nice acronyn for the wrong word? What about 1 letter abbreviations for 'J' for adjacency (aka break Junction), and abundance 'A' (or would C/copies work better)?

The main problem with ADJ + ABU/ABD is that they start with the same letter and require some mental effort to distinguish. I think “adjacency” + “quantity” is the perfect word pair, and can be translated into either of those choices:
1. **ADJ + QTY + BOTH**
1. ADJ + QTY + ADJQTY
1. A + Q + B
1. A + Q + AQ

To me, the first one looks like it's the most elegant and readable.